src/openms/source/ANALYSIS/OPENSWATH/MRMAssay.cpp

```diff
@@ -185,9 +185,9 @@ namespace OpenMS
     OpenMS::MRMIonSeries mrmis;
 
     size_t j = 0;
-    for (size_t i = 0; i < exp.getTransitions().size(); ++i)
+    for (Size tr_it = 0; tr_it < exp.getTransitions().size(); ++tr_it)
```

Comment:
@cbielow ok, I will try to change this. It is just that single letter variables are hard to search and replace, therefore I also dont like `j`